themes/classic/templates/_partials/header.tpl

```diff
@@ -1,30 +1,24 @@
 {block name='header_nav'}
   <nav class="header-nav row">
     <div class="container">
-      <div class="col-lg-8 col-lg-offset-7 col-md-10 col-md-offset-4">
         <div class="row">
           {hook h='displayNav'}
         </div>
-      </div>
     </div>
   </nav>
 {/block}
 
-{block name='header_logo'}
-<div class="row">
-  <div class="container">
-    <div class="col-lg-8 col-md-6">
-      <img class="logo img-responsive" src="{$shop.logo}" alt="logo" width="180" />
-    </div>
-    <div class="col-md-1 col-md-offset-1">{hook h="displayHeaderMiddle"}</div>
-  </div>
-</div>
-{/block}
-
 {block name='header_top'}
-  <div class="header-top row">
-    <div class="container _relative navigation">
-      {hook h='displayTop'}
+    <div class="container">
+      <div class="header-top row _margin-bottom-medium">
+        <div class="col-md-2">
+          <a href="{$urls.base_url}" title="{l s='Homepage'}" >
+            <img class="logo img-responsive" src="{$shop.logo}" alt="logo" />
```

Comment:
Maybe {$shop.name} {l s='logo'}? I see that you guys using aria and other WCAG stuff, in that case each element should be described "as is" which means that this element is... "Some Store logo" :) But I think it is something what should be discussed